docs/QUICKSTART.md

```diff
@@ -1,76 +1,190 @@
-- Install eraser from [deployment YAML](https://github.com/Azure/eraser/blob/v0.1.0/deploy/eraser.yaml).
-
-```bash
-$ kubectl apply -f https://raw.githubusercontent.com/Azure/eraser/v0.1.0/deploy/eraser.yaml
-namespace/eraser-system created
-customresourcedefinition.apiextensions.k8s.io/imagejobs.eraser.sh created
-customresourcedefinition.apiextensions.k8s.io/imagelists.eraser.sh created
-serviceaccount/eraser-controller-manager created
-role.rbac.authorization.k8s.io/eraser-leader-election-role created
-clusterrole.rbac.authorization.k8s.io/eraser-manager-role created
-rolebinding.rbac.authorization.k8s.io/eraser-leader-election-rolebinding created
-clusterrolebinding.rbac.authorization.k8s.io/eraser-manager-rolebinding created
-configmap/eraser-manager-config created
-deployment.apps/eraser-controller-manager created
-
-$ kubectl get pods -n eraser-system
-NAME                                         READY   STATUS    RESTARTS   AGE
-eraser-controller-manager-759c7c8794-q8qqg   1/1     Running   0          22s
-```
-
-- Create an [ImageList](./test/e2e/test-data/eraser_v1alpha1_imagelist.yaml) and specify the images you would like to remove.
-
-Example:
-```bash
-cat <<EOF | kubectl apply -f -
-apiVersion: eraser.sh/v1alpha1
-kind: ImageList
-metadata:
-  name: imagelist
-spec:
-  images:
-    - sha256:2834dc507516af02784808c5f48b7cbe38b8ed5d0f4837f16e78d00deb7e7767
-    - docker.io/library/nginx:latest
-    - redis
-EOF
-```
+# Quickstart Tutorial
+
+1. Clone or create a new Codespace from the Eraser repo
+
+    ```shell
+    git clone https://github.com/Azure/eraser.git
+    cd eraser
+    ```
+    
+2. Create a cluster
+
+    This will install and create a Kind cluster with a control-plane and two worker nodes.
+
+    ``` shell
+    curl -Lo ./kind https://kind.sigs.k8s.io/dl/v0.13.0/kind-linux-amd64
+    chmod +x ./kind
+    sudo mv ./kind /usr/bin
+    kind create cluster --config test/e2e/kind-config.yaml
+    ```
+    
+    Verify your cluster is running.
+  
+    ```shell
+    $ kubectl cluster-info --context kind-kind
+    Kubernetes control plane is running at https://127.0.0.1:40989
+    CoreDNS is running at https://127.0.0.1:40989/api/v1/namespaces/kube-system/services/kube-dns:dns/proxy
+
+    To further debug and diagnose cluster problems, use 'kubectl cluster-info dump'.
+    ``` 
+
+
+3. Deploy Eraser to the Cluster
+
```

Comment:
**No change needed** just wanted to call out this will be replaced with a Helm install details when Helm chart is created & hosted & accessible

Reply:
Thanks for the comment. Will update when the helm chart is ready

Reply:
Added a link to the Eraser Helm chart readme in https://github.com/Azure/eraser/pull/186/commits/a26a160733e26313d0636cdb27aee4d7f1fefb4c. The link points to readme being merged in #191, so it will not work until that is in.